scribus/ui/smcstylewidget.ui

```diff
@@ -427,20 +412,69 @@
        </layout>
       </item>
       <item>
-       <spacer>
-        <property name="orientation">
-         <enum>Qt::Vertical</enum>
-        </property>
-        <property name="sizeType">
-         <enum>QSizePolicy::MinimumExpanding</enum>
-        </property>
-        <property name="sizeHint" stdset="0">
-         <size>
-          <width>300</width>
-          <height>0</height>
-         </size>
-        </property>
-       </spacer>
+       <layout class="QHBoxLayout" name="horizontalLayout">
+        <item>
+         <widget class="QLabel" name="label">
+          <property name="text">
+           <string>Smallest Word</string>
```

Comment:
The name is not very clear, since nothing tells that is is about hyphenation. I’d add a new “Hyphenation” group similar to “Advanced Formatting”, since I expect at least two new hyphenation settings.